node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -187,11 +187,6 @@ impl<I: NodeIdT, C: Context + 'static> Synchronizer<I, C> {
             .add(sender, pvv, now);
     }
 
-    /// Returns the timestamps at which we are supposed to add cached vertices.
-    pub(crate) fn timestamps_to_add_vertices(&self) -> Vec<Timestamp> {
-        self.vertices_to_be_added_later.keys().cloned().collect()
-    }
```

Comment:
Why is this not needed anymore? Doesn't seem like something that would be here b/c of the old joiner <-> EraSupervisor connection.

Reply:
That was only used in `recreate_timers`, which is now unneeded, because the only reason for its existence was that the timers were lost in the transition from joiner to validator.